uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/QuickStartsTest.java

```diff
@@ -115,136 +143,29 @@ public void setupDeleteProjects() {
-		MavenImportWizard.importProject(quickstart);
-		assertTrue("There are some errors in Error Log", LogGrapper.getPluginErrors("fuse").size() == 0);
-		CamelProject project = new CamelProject("beginner-camel-errorhandler");
-		project.enableCamelNature();
-		FuseServerManipulator.addModule(serverRequirement.getConfiguration().getServer().getName(), "beginner-camel-errorhandler");
-		try {
-			new WaitUntil(new FuseLogContainsText("(CamelContext: errors-example-context) started"));
-		} catch (WaitTimeoutExpiredException e) {
-			fail("Project 'beginner-camel-errorhandler' was not sucessfully deployed!");
-		}
-		checkProblemsView();
+	@After
+	public void setupRemoveAllDeployedProjects() {
+		FuseServerManipulator.removeAllModules(serverRequirement.getConfiguration().getServer().getName());
 	}
 
 	/**
 	 * <p>
-	 * Test tries to deploy 'beginner-camel-log' quickstart to Red Hat Fuse Runtime.
+	 * Test tries to import a quickstart from Red Hat Fuse Runtime.
 	 * </p>
 	 * <b>Steps:</b>
 	 * <ol>
-	 * <li>import 'beginner-camel-log' project from Red Hat Fuse quickstarts</li>
+	 * <li>import a project from Red Hat Fuse quickstarts</li>
 	 * <li>check that project is ok (no errors, unresolved dependencies, ...)</li>
-	 * <li>deploy the project to Red Hat Fuse</li>
-	 * <li>check whether the project is successfully deployed (check whether Red Hat Fuse log contains '(CamelContext:
-	 * log-example-context) started')</li>
 	 * </ol>
 	 */
 	@Test
-	public void testBeginnerCamelLog() {
-
-		String quickstart = serverRequirement.getConfiguration().getServer().getHome() + "/quickstarts/beginner/camel-log";
-		MavenImportWizard.importProject(quickstart);
-		assertTrue("There are some errors in Error Log", LogGrapper.getPluginErrors("fuse").size() == 0);
-		CamelProject project = new CamelProject("beginner-camel-log");
-		project.enableCamelNature();
-		FuseServerManipulator.addModule(serverRequirement.getConfiguration().getServer().getName(), "beginner-camel-log");
-		try {
-			new WaitUntil(new FuseLogContainsText("(CamelContext: log-example-context) started"));
-		} catch (WaitTimeoutExpiredException e) {
-			fail("Project 'beginner-camel-log' was not sucessfully deployed!");
-		}
-		checkProblemsView();
-	}
+	public void testQuickStart() {
 
-	private void checkProblemsView() {
-		
-		List<Problem> problems = new ProblemsView().getProblems(ProblemType.ERROR);
-		if (problems.size() == 0) {
-			return;
-		}
-		if (problems.size() == 1 && problems.get(0).getDescription().startsWith("Referenced file contains errors (http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd).")) {
-			throw new JiraIssue("ENTESB-6115");
-		}
-		fail("There are some errors in Problems view");
+		MavenImportWizard.importProject(quickstart, TimePeriod.getCustom(1000));
+		new CamelProject(new ProjectExplorer().getProjects().get(0).getName()).update();
+		assertTrue("Error log contains some fuse tooling errors", LogChecker.noFuseError());
```

Comment:
realizing just now that there is also another util method: LogChecker.assertNoFuseError() whcih can replace the full line (sorry)